tests/test_LocalClassifierPerNode.py

```diff
@@ -189,10 +350,82 @@ def test_predict_sparse(fitted_logistic_regression):
     assert_array_equal(ground_truth, prediction)
 
 
+def test_predict_proba(fitted_logistic_regression):
```

Comment:
maybe you can use parametrize to reduce redundancy when tests are the same in different files